docs/src/main/sphinx/object-storage/file-system-s3.md

```diff
@@ -42,11 +42,13 @@ support:
 * - `s3.sse.type`
   - Set the type of S3 server-side encryption (SSE) to use. Defaults to `NONE`
```

Comment:
Please add to the description of the PR that it relates to https://github.com/trinodb/trino/pull/22992